### PR TITLE
[Website] 0.96 Release stuff

### PR DESCRIPTION
**Pull request by achow101**
This contains the release post for 0.96, the contributors page update, and todo list update.

The release post is not finished yet. It needs to have the correct download links which are only available once the binaries are posted. Let me know when that happens and I will update it.

To those of you who see this PR and want to post it to reddit, bitcointalk, etc., please don't. Wait for the binaries to be posted and for goatpig to make the announcement on bitcointalk before posting elsewhere about this release. Also, only the developers of this project should really be posting release announcements, so if you aren't one, I ask that you don't.

### CODE REVIEW

**Comment by achow101** [comment]
This should be ready to merge now.

File: _posts/2017-05-01-0.96.0-release.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+### Migrating to A New Wallet Format
+
+Armory 0.96.0 marks the beginning to a new wallet format written in C++ and uses LMDB. Currently this wallet format is only a mirrored version of the current Python wallet format. It only contains public data and metadata in order to allow the C++ signing code to function, including support for P2SH-P2PK and P2SH-P2WPKH outputs which are not supported by the current Python wallet format. In the future, this new format will support BIP 32 and compressed public keys.
```
Comment (goatpig) [inline _posts/2017-05-01-0.96.0-release.md]: mirrored wallets already support compressed pub keys. all the p2sh scripts use compressed keys

File: _posts/2017-05-01-0.96.0-release.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+### Dynamic Transaction Fees
+
+In Armory 0.95.0, we added the option to use a fee per byte given by the bitcoind running behind the scenes. This transaction fee option has been expanded to allow users to set their own confirmation target, set a different fee rate, or set a fixed transaction fee. The automatic fee per byte option is selected by default and will adjust the transaction fee based upon the inputs selected for your transaction.
+
+### Replace-By-Fee Support
+
+When sending a transaction, the user can now choose to mark a transaction as RBF enabled so that its fee can be increased while it is unconfirmed. While a transaction is unconfirmed, it's fee can be increased with the Fee Bump option. This will create a RBF transaction for transactions sent by your wallet. The fee bump will never be greater than 10% of the original fee value.
```
Comment (goatpig) [inline _posts/2017-05-01-0.96.0-release.md]: the rbf feature from the ledger does not bump the fee per se, it spawns the send dialog prefilled with the ledger tx inputs and outputs. the user is left free to modify the fee before sending

File: _posts/2017-05-01-0.96.0-release.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+# Configuration files
+
+Config files can now be created for ArmoryQt (armoryqt.conf) and ArmoryDB (armorydb.conf). These files can take any valid command line argument, one per line. Arguments can be commented out using a `#` (pound) symbol. The command line arguments used will always override the arguments set in the config file.
+
+
```
Comment (goatpig) [inline _posts/2017-05-01-0.96.0-release.md]: should add a note about translations as well

**Pull request merged**